frontend/src/components/client/controls/ClientSearch.tsx

```diff
@@ -0,0 +1,28 @@
+import { SelectItem } from '../../../models';
+import { BaseSelect } from '../../controls/form-controls/select/BaseSelect';
+
+export type ClientFilterOption = {
+    label: string,
```

Comment:
Heb je de editorconfig plugin? Please install
Dan is de layout overal hetzelfde...